src/main/api-utilities/VerifierInterface.js

```diff
@@ -0,0 +1,21 @@
+export class VerifierInterface {
+    constructor(scenario, type) {
+        if (scenario && type) {
+            this.scenario = scenario;
+            this.type = type;
+        } else {
+            throw new Error('Scenario and Type must be defined');
```

Comment:
we can change because sometimes we can define one scenario and dont the type, the error have to be scenario or type